blazor/tooltip/content.md

```diff
@@ -11,31 +11,115 @@ documentation: ug
 
 A text or a piece of information assigned to the Tooltip's `Content` property will be displayed as the main text stream of the Tooltip. It can be a string or a template content. If the `Content` property is not provided with any specific value, then it takes the value assigned to the attribute of the target element on which the Tooltip was initialized. The content can also dynamically be assigned to the Tooltip via AJAX.
 
-## Template content
+##  Displaying dynamic HTML Content
 
-Any text or image can be added to the Tooltip, by default. To customize the Tooltip layout or to create your own visualized element on the Tooltip, template can be used.
+Any type of formatted content can be displayed within the Tooltip by default. To enhance the Tooltip layout or create a custom visualized element, use the `<ContentTemplate>` tag to define the structure.
```

Comment:
By default, the Tooltip can show any kind of formatted content. To improve its layout or create a custom visual element, utilize the `<ContentTemplate>` tag to establish the desired structure. You can also use @(MarkupString) to dynamically render HTML content.

Reply:
Content changed as per suggested.